repository/Seaside-Core.package/WASessionCookieProtectionFilter.class/README.md

```diff
@@ -0,0 +1,7 @@
+The session cookie protection filter ensures that the wrapped request handler only accepts requests from the same browser session. This filter is specifically useful to protect session hijacking when using the (default) query field session tracking strategy.
+
+Because WAQueryFieldHandlerTrackingStrategy puts the Seaside session key in the url, a session can be easily hijacked by copying the url. This request filter prevents this by requiring a browser session cookie associated to the Seaside session. As a result, a copied Seaside url can only be used in the same browser session.
```

Comment:
I find the word "browser session" confusing. Is that an established term for all tabs and windows of a browser. Would "browser instance" be better or equally confusing.

Reply:
Here's where it's coming from: we use a 'browser session cookie'. So, that's why we refer to it as a 'browser session'